web/styles/app_variables.css

```diff
@@ -300,6 +300,15 @@
                     ) - var(--length-line-oversize-block)
             ) / 2
     );
+    /* When using rem in the past, the root font size was 16px when the
+       base font-size was 14px. Now, we have two modes, one with 14px
+       base font-size and one with 16px base font-size. We multiply by
+       16 / 14 so rem equivalent value will remain the same in the
+       legacy 14px mode, while scaling the size up for 16px mode.
+       This value is not to be used anywhere rem is currently in use,
+       but instead this should be used on a case by case basis.
+       */
+    --rem-intermediate-unit: calc(16 / 14 * var(--base-font-size-px));
```

Comment:
Yeah, let's please not do this. Figure out the pixel value at 16px/1rem, and then calculate that as an em value at 14px/1em. Just note it in the CSS and use the em value directly. The beauty of ems is that they scale up as the font-size changes, without having to get into all the preciousness of `calc()`

Reply:
From the commit message:

> The original plan was to use em instead of rem and multiply it by
(16/14 * 0.8), but since em depends on the parent element, there was a
case in the poll button widget where 1em was equal to 16px in the dense
14px mode. While, the right thing to do might have been some refactor to
make it work as desired, the safest thing to do right now might be to
use the --base-font-size-px variable directly for the 9.0 release.

The issue faced with EMs is that the em value will depend on the parent element font-size. 

In case of modal content with class `.modal__content`, we set the font-size to `1.1429em` which is equal to 16px when using the legacy dense mode. This means that when using the poll create button, the em value for the trash icon will be 16px, when we want that value to be 14px. A working `rem` unit would be really great to have, but until that happens, I don't have confidence of em working for `.small` font-size with changing just that one line, since the em value can be different for different components for all `.small` occurences.

A way to always have the same `em` value for `.small` button would be to set the font-size in `.new-style` selector, but in that case, we will end up actually changing the font-size for a lot of a content. This is because we `.new-style` with divs that might have other stuff inside which may not like it's font-size being determined by `.new-style` class.